frontend/src/components/Calendar/FirstLastDateRow.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+
+import { ClickableDateContainer } from './ClickableDateContainer'
+import { EmptyDateContainer } from './EmptyDateContainer'
+
+export const FirstLastDateRow = (
+  firstDayOfThisMonth: number,
+  lastDateOfPreviousMonth: number,
+  firstRow: boolean,
+  assignedMonth: number,
+  eventDates: number[],
+) => {
+  let newMonthDate = 1
+  let firstDateToPut = lastDateOfPreviousMonth - firstDayOfThisMonth + 2
```

Comment:
what does firstDateToPut mean

Reply:
We meant to represent the first date to put on the first/last row of the calendar! 

But anyways, the file has been removed with new DateRows component